feature-group-definitions/flexbox-gap.yml

```diff
@@ -1,2 +1,3 @@
+name: "Flexbox gap"
```

Comment:
https://caniuse.com/flexbox-gap calls this "gap property for Flexbox" but https://web.dev/articles/bridging-the-gap#flexbox_gap is an example of "Flexbox gap" as a shorter form.